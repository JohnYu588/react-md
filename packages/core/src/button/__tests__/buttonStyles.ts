import { button } from "../buttonStyles.js";

describe("button", () => {
  it("should be callable without any arguments", () => {
    expect(button()).toMatchSnapshot();
  });
});
