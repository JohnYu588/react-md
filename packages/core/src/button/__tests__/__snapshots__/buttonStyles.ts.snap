// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button should be callable without any arguments 1`] = `"rmd-button rmd-button--text"`;
